conf/test_route_control.py

```diff
@@ -227,14 +256,20 @@ def test_given_valid_new_route_when_add_new_route_entry_and_mac_known_then_route
             prefix_len=24,
         )
         self.route_controller.add_new_route_entry(route_entry)
+        self.mock_bess_controller.add_route_to_module()
         self.mock_bess_controller.add_route_to_module.assert_called_once()
 
     def test_given_valid_new_route_when_add_new_route_entry_and_mac_known_and_neighbor_not_known_then_update_module_is_created_and_modules_are_linked(
         self,
```

Comment:
```suggestion
    @patch("conf.route_control.send_ping")
    def test_given_valid_new_route_when_add_new_route_entry_and_mac_known_and_neighbor_not_known_then_update_module_is_created_and_modules_are_linked(
        self, _
```
This should fix the permission denied issue.